src/com/photoncat/aiproj2/heuristic/Heuristic2.java

```diff
@@ -0,0 +1,902 @@
+                continue;
+            }
+            //live2 11*0
+            if (    getByDirLen(board,row,col,i,-1)==cur&&
+                    getByDirLen(board,row,col,i,-2)==cur){
+
+                if (getByDirLen(board,row,col,i,1)==0){
+                    score+=750;     //2111*0
+                    if (getByDirLen(board,row,col,i,-3)==0){
+                        score+=39250;  ////0111*0
+                        if (player==PieceType.CIRCLE){
+                            score+=50000;
+                        }
+                    }
+                }
+                if ((   getByDirLen(board,row,col,i,1)==opp||
+                        getByDirLen(board,row,col,i,1)==-1)&&
+                        getByDirLen(board,row,col,i,-3)==0){
+                    score+=500;      //0111*2
+                }
+
+                continue;
+            }
+            //dead 2 1*1
+            if (    getByDirLen(board,row,col,i,-1)==cur&&
+                    getByDirLen(board,row,col,i,1)==cur) {
+
+                score += 600;
+                if (getByDirLen(board, row, col, i, -2) == 0 &&
+                        getByDirLen(board, row, col, i, 2) == 0) {
+                    score += 25000;
+                    if (player==PieceType.CIRCLE){
+                        score+=50000;
+                    }
+                    continue;
+                }
+                if ((getByDirLen(board, row, col, i, -2) == opp ||
+                        getByDirLen(board, row, col, i, -2) == -1) && (
+                        getByDirLen(board, row, col, i, 2) == opp ||
+                                getByDirLen(board, row, col, i, 2) == -1)) {
+                    continue;
+                } else {
+                    score += 700;
+                    continue;
+                }
+            }
+            if (getByDirLen(board, row, col, i, -1) == cur &&
+                    getByDirLen(board, row, col, i, -2) == 0
+                    && getByDirLen(board, row, col, i, 1) == 0
+                    ) {
+                one++;
+            }
+            int lonewolf=0;
+            for(int k=-4;k<=0;k++){
+                int temp=0;
+                for (int l = 0; l <= 4; l++) {
+                    if (getByDirLen(board,row,col,i,k+l)==cur){
+                        temp++;
+                    }
+                    else {
+                        if (    getByDirLen(board,row,col,i,k+l)==opp||
+                                getByDirLen(board,row,col,i,k+l)==-1)
+                            temp=0;
+                        break;
+                    }
+                }
+                lonewolf+=temp;
+            }
+            score+=lonewolf*15;
+
+
+        }
+        if (one>=2){
+            score+=20000;
+            if (player==PieceType.CIRCLE){
+                score+=500;
+            }
+        }
+        return score;
+    }
+
+    private int getByDirLen(Board board, int row, int col, int dir, int len) {
```

Comment:
You should use an enum instead of int for a value like the direction. Enum types comment themselves, while the caller of getByDirLen ( this happens 100+ times ) don't. 
Enum types also block common errors - For example, How will your code work when dir == 9?